test/parallel/test-fs-read-file-sync-hostname.js

```diff
@@ -1,10 +1,14 @@
 'use strict';
-require('../common');
-var assert = require('assert');
-var fs = require('fs');
+const common = require('../common');
+const assert = require('assert');
+const fs = require('fs');
 
-// test reading from hostname
-if (process.platform === 'linux2') {
-  var hostname = fs.readFileSync('/proc/sys/kernel/hostname');
-  assert.ok(hostname.length > 0);
+if (!common.isLinux) {
```

Comment:
`common.isLinux` checks if `process.platform === 'linux'`, but this test was checking for `linux2`. Is that planned?